tracing-honeycomb/examples/async_tracing.rs

```diff
@@ -90,12 +90,11 @@ fn register_global_subscriber() {
         transmission_options: libhoney::transmission::Options::default(),
     };
 
-    // TODO: helper fn for this exported by honeycomb tracing lib
     let telemetry_layer = mk_honeycomb_tracing_layer("async-tracing_example", honeycomb_config);
 
     let subscriber = telemetry_layer // publish to tracing
         .and_then(tracing_subscriber::fmt::Layer::builder().finish()) // log to stdout
-        .and_then(LevelFilter::INFO) // omit low-level debug tracing (eg tokio executor)
+        .and_then(LevelFilter::INFO) // filter out low-level debug tracing (eg tokio executor)
         .with_subscriber(registry::Registry::default()); // provide underlying span data store
```

Comment:
Nit: I'd suggest writing this instead as:

```rust
let subscriber = registry::Registry::default() // provide underlying span data store
    .with(LevelFilter::INFO) // filter out low-level debug tracing (eg tokio executor)
    .with(tracing_subscriber::fmt::Layer::builder().finish()) // log to stdout
    .with(telemetry_layer); // publish to tracing
```

It makes the hierarchy _somewhat_ cleaner and closer to other frameworks.